.github/company.js

```diff
@@ -0,0 +1,128 @@
+let querySymbol = new URLSearchParams(window.location.search);
+let symbol = querySymbol.toString()
```

Comment:
This is not how you should use the URLSearchParams() object, see here: 
https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#examples